src/pages/components/video-player.mdx

```diff
@@ -29,6 +30,15 @@ The Video player aspect ratio should match that of the video, or use one of the
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Notes / Options |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Video ID | Short copy | Yes | 1 | - |
+| Caption | Short copy | No | 1 | Could be provided through video source |
+| Show caption | True/False | Yes | 1 | True = Show caption<br/>False = Don't show caption |
+| Autoplay | True/False | Yes | 1 | True = Autoplay<br/>False = Don't autoplay (default) |
+| Aspect ratio | Option list | Yes | 1 | Default is 16x9. Other ratios are 9x16, 2x1, 1x2, 4x3, 3x4, 1x1 |
```

Comment:
I think going off of the last comment I think there is also something else at play here. These are more functional, and code property-related than content model related, right? To me, the content is the thumbnail, thumbnail alt, and caption. Or am I not looking at this right?

Reply:
The user I'm looking at it from is someone who would utilize this information when adding the component through a CMS. So the user might not need to add a thumbnail cause that comes from Kaltura, but they would need to add the Video ID. They could also in the CMS change the autoplay function.
I'm not sure on the aspect ratio, but I saw it called out in some documentation so I added it, however I would be fine to get rid of it considering that is probably designated by Kaltura.

Reply:
cc: @JennySanchez @wonilsuhibm @mfearer94 

So I think this is something we need to be careful with here. Who is the user of the website? Is it for CMS users, or for adopters of our design system? Is it the designer or is it the developer? I'm not saying it can't be all the above even, but I'm struggling a little between this functional or even technical model, vs the content model, and then even vs. the CMS model if that makes sense. I don't think this discussion should block this from going in, but I think we need to have a solid point of view and direction around this as we make content for the website.

Reply:
100% agree @photodow. We should discuss this at our next design review if possible.